plugins/vst_base/RemoteVstPlugin.cpp

```diff
@@ -2251,6 +2559,13 @@ int main( int _argc, char * * _argv )
 		}
 	}
 
+#ifdef NATIVE_LINUX_VST
+	if (EMBED)
+	{
+		cerr << "Native linux VST works only without embeding." << endl;
```

Comment:
Have you tested this?

Reply:
Yes, you can't open or close plugin window then.